Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+click = "*"
+numpy = "*"
+cffi = "*"
+scipy = "*"
+astropy = "*"
+corner = "*"
+pytest = "*"
```

Comment:
I feel like these deps should be in `dev-packages`?
Also, should `cosmoHammer` be on this list somewhere?

Reply:
Ah yes, I was clumsy with what is dev and what is not...
For some reason I cannot attach new Pipfile, so here is the [link](https://www.dropbox.com/s/yqr0evy8rhcqo2s/pipfile?dl=0).

The only package needed for non-dev installation should be `21CMMC`, but I've also put `21cmfast`, as `pipenv` has some bug when reading links in `setup.py` and the package should also be editable (I think the two are connected).

After installation with this new Pipfile (so `pipenv install --dev`) here is an output of [`pipenv graph`](https://www.dropbox.com/s/yq565w6pw28srtj/pipenv_graph.txt?dl=0), so `cosmoHammer` and others are under `21CMMC`.

Reply:
OK I think I kind of understand: `pipenv` is using `setup.py` to get the deps of the package. But doesn't that just make `pipenv` redundant? Isn't the point of `pipenv` to be able to specify an exact environment (whereas requirements in `setup.py` specify unpinned requirements? 

I'm sorry I'm not up to speed with this! In the end, as long as it works for you and doesn't add extra complexity, I'm OK with it!

Reply:
One can think of `pipenv` as `pip`, with addition that it creates virtual environment, manages all the venvs and strictly checks dependency compatibilities. 
I actually haven't realised at first how nicely have you packed everything, so except for easily creating dev environment, it is redundant.

Reply:
I'm happy for it still to be there just for the dev environment. Doesn't it do stuff like automatically update your requirements when you update packages? That seems useful enough.

Reply:
Yes, it does that! OK, great :)